src/app/core/browse/browse-entry-search-options.model.ts

```diff
@@ -0,0 +1,17 @@
+import { PaginationComponentOptions } from '../../shared/pagination/pagination-component-options.model';
+import { SortOptions } from '../cache/models/sort-options.model';
+
+/**
+ * A class that defines the search options to be used for fetching browse entries or items
+ * - metadataDefinition:  The metadata definition to fetch entries or items for
+ * - pagination:          Optional pagination options to use
+ * - sort:                Optional sorting options to use
+ * - scope:               An optional scope to limit the results within a specific collection or community
+ */
+export class BrowseEntrySearchOptions {
+  constructor(public metadataDefinition: string,
+              public pagination?: PaginationComponentOptions,
+              public sort?: SortOptions,
+              public scope?: string) {
+  }
```

Comment:
Yes, starts-with is still missing. The browse tasks were divided in to very small chunks in order to have easy tasks for new angular devs during the sprints. So supporting starts-with on browse pages was [a ticket on its own](https://github.com/DSpace/dspace-angular/issues/248).

@Atmire-Kristof is working on that as we speak, and it will arrive in a separate PR soon.